ticket.go

```diff
@@ -59,13 +59,20 @@ func OpenStore(r io.Reader) (*Store, error) {
 	return s, nil
 }
 
-// Yada.
+// WriteTo writes encodes data to JSON and writes to file.
 func (s *Store) WriteTo(w io.Writer) error {
-	w = s.tickets
-	for _, tk := range w{
-		json.NewEncoder(tk).Encode
+	tks := s.tickets
+
+	for _, tk := range tks {
+		//encoder, err := json.Marshal(tk)
+		err := json.NewEncoder(w).Encode(tk)
```

Comment:
Although there's nothing wrong with creating a new Encoder for each ticket, we don't really need to do that, do we? We could create a single Encoder on `w` before the loop (let's call it `encoder`), and then we could call `encoder.Encode(tk)` each time through the loop.